lib/verifier.js

```diff
@@ -101,7 +101,7 @@ Verifier.checkTxProposalBody = function(credentials, txp) {
     var proposalHeader = {
       outputs: outputs,
       message: txp.encryptedMessage || txp.message || null,
-      payProUrl: txp.payProUrl || undefined
+      payProUrl: txp.payProUrl
```

Comment:
this will produce different hashes for headers with ``null`` and ``undefined`` payProUrl. Does it make sense? Wouldn't it be better to make them equal (by nullifying undefined value)?